src/main/java/org/sagebionetworks/bridge/services/AppService.java

```diff
@@ -342,7 +346,20 @@ public App createApp(App app) {
             throw new EntityAlreadyExistsException(App.class, IDENTIFIER_PROPERTY, app.getIdentifier());
         }
         
-        subpopService.createDefaultSubpopulation(app);
+        Study study = Study.create();
+        study.setAppId(app.getIdentifier());
+        study.setIdentifier(app.getIdentifier() + "-study");
+        study.setName("Initial Study");
```

Comment:
I don't know. I'm okay with that, because it's easily changed or deleted. In many cases one app may still have one study and it would make sense. Updated. Also added tests to AppServiceTest to verify the creation of study and Sage as a default organization.